src/Microsoft.AspNetCore.Server.HttpSys/HttpSysOptions.cs

```diff
@@ -2,18 +2,21 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Globalization;
 using Microsoft.AspNetCore.Http.Features;
 
 namespace Microsoft.AspNetCore.Server.HttpSys
 {
     public class HttpSysOptions
     {
+        private const long DefaultRejectionVerbosityLevel = (long)Http503ResponseVerbosityLevel.Basic; // Http.sys default.
```

Comment:
Use the enum type everywhere. You only need to cast it to a long inside of SetRejectionVerbosity and maybe not even then.

Reply:
Resolved in next iteration